earthaccess/auth.py

```diff
@@ -11,22 +12,34 @@
 
 from .daac import DAACS
 
+try:
+    user_agent = f"earthaccess v{importlib.metadata.version('earthacess')}"
+except importlib.metadata.PackageNotFoundError:
+    user_agent = "earthaccess"
+
+
 logger = logging.getLogger(__name__)
 
 
 class SessionWithHeaderRedirection(requests.Session):
     """
     Requests removes auth headers if the redirect happens outside the
     original req domain.
-    This is taken from https://wiki.earthdata.nasa.gov/display/EL/How+To+Access+Data+With+Python
     """
 
-    AUTH_HOST = "urs.earthdata.nasa.gov"
+    AUTH_HOSTS: List[str] = [
+        "urs.earthdata.nasa.gov",
+        "cumulus.asf.alaska.edu",
+        "sentinel1.asf.alaska.edu",
+        "datapool.asf.alaska.edu",
+    ]
 
     def __init__(
         self, username: Optional[str] = None, password: Optional[str] = None
     ) -> None:
         super().__init__()
+        self.headers.update({"User-Agent": user_agent})
```

Comment:
`earthaccess` will now use the `User-Agent` header to identify requests to EDL. 🎉